sql/core/src/test/scala/org/apache/spark/sql/sources/CatalystScanSuite.scala

```diff
@@ -0,0 +1,73 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Just FYI, I opened a issue in scalastyle - `https://github.com/scalastyle/scalastyle/issues/252`.